lib/testScenario/postmanCollectionGenerator.ts

```diff
@@ -49,6 +54,12 @@ export class PostmanCollectionGenerator {
     }
     let index = 0;
     const runId = generateRunId();
+    if (this.opt.markdownReportPath) {
```

Comment:
markdownReportPath is required?

Reply:
better to set default markdownReportPath path in outputFolder.

Reply:
marked the param as optional